microsoft-edge/webview2/get-started/winforms.md

```diff
@@ -68,71 +82,98 @@ Use NuGet to add the WebView2 SDK to the project.
+    :::image type="complex" source="./media/winforms-hello-webview.png" alt-text="hello webview" lightbox="./media/winforms-hello-webview.png":::
+       hello webview  
+    :::image-end:::
 
-## Step 4 - Handle Window Resize Events  
+    > [!NOTE]
+    > If you are working on a high DPI monitor, you may have to [configure your Windows Forms app for high DPI support][DotnetFrameworkWinformsHighDpiSupportWindowsFormsConfiguringYourWindowsFormsAppForHighDpiSupport].  
+    
+## Step 4 - Add controls and process window resize events  
 
-Add a few more controls to your Windows Forms from the toolbox, and then handle window resize events appropriately.  
+Add more controls to your Windows Forms from the toolbox, then process window resize events.  
 
-1.  In the **Windows Forms Designer**, open the **Toolbox**.  
-1.  Drag and Drop a **TextBox** into the Windows Forms App.  Name the **TextBox** `addressBar` in the **Properties Tab**.  
-1.  Drag and Drop a **Button** into the Windows Forms App.  Change the text in the **Button** to `Go!` and name the **Button** `goButton` in the **Properties Tab**.  
-    
-    The app should look like the following image in the designer.  
+1. Choose **View** > **Toolbox**.
+1. In the **Toolbox** choose **Common Controls**. 
+1. Drag and drop **TextBox** control into the Windows Forms App.
+1. In the Properties panel, change the **\(Name\)** to **addressBar**.  
+1. Drag and Drop a **Button** control into the Windows Forms App.
+1. In the Properties panel, change the **\(Name\)** to **goButton**.
+1. Change the **Text** property to **Go!**.
+1. Resize the button as needed to display the text.
+1. Arrange the text box to the left of the button, aligned by the text as shown. 
     
     :::image type="complex" source="./media/winforms-designer.png" alt-text="WinForms designer" lightbox="./media/winforms-designer.png":::
        WinForms designer  
     :::image-end:::  
 
-1.  In the `Form1.cs` file, define `Form_Resize` to keep the controls in place when the App Window is resized.
-    
-```csharp
-public Form1()
-{
-    InitializeComponent();
-    this.Resize += new System.EventHandler(this.Form_Resize);
-}
+1. Resize the text box as shown.
 
-private void Form_Resize(object sender, EventArgs e)
-{
-    webView.Size = this.ClientSize - new System.Drawing.Size(webView.Location);
-    goButton.Left = this.ClientSize.Width - goButton.Width;
-    addressBar.Width = goButton.Left - addressBar.Left;
-}
-```  
+    :::image type="complex" source="./media/winforms-designer-txtbtn.png" alt-text="WinForms designer textbox and button" lightbox="./media/winforms-designer-txtbtn.png":::
+       WinForms designer textbox and button
+    :::image-end::: 
+
+1. Choose **View** > **Code** to open the `Form1.cs` file.
+
+    Define `Form_Resize` to keep the controls in place when the App Window is resized.
```

Comment:
lowercase as "app window"?  If title bar shows **App Window** label, maybe use bold with casing like the UI.